spec/features/job_seekers_can_search_vacancies_spec.rb

```diff
@@ -217,3 +217,26 @@
     end
   end
 end
+
+RSpec.feature 'Searching vacancies by subject', js: true do
```

Comment:
You can add the `js: true` to the context block that you want javascript testing and keep it within the previous describe block, rather than having a separate describe block for it.

Reply:
Thanks, Resolved in e5c1f5e